packages/manager/apps/telecom/src/app/telecom/pack/move/address/move-address.controller.js

```diff
@@ -0,0 +1,43 @@
+import { UNBUNDLING } from '../pack-move.constant';
+
+export default class MoveAddressCtrl {
+  /* @ngInject */
+  constructor($scope) {
+    this.$scope = $scope;
+  }
+
+  $onInit() {
+    this.portLineNumber = false;
+    this.keepLineNumber = false;
+  }
+
+  /**
+   * Check if can keep line number
+   * @returns {boolean}
+   */
+  canKeepLineNumber() {
```

Comment:
i don't think the keepLineNumber variable is useful here.
also it's bad practice to mutate an object in a method that is supposed to returns a value

why not simply returning canKeep ?

Reply:
This variable is very useful, if canKeep is false, the user couldn't change the checkbox keepLineNumber and then we are sure that the keepLineNumber is set to false and will not generate some errors in next step when the user validates move. The canKeep method defines if the checkbox is enabled and the keepLineNumber variable is the value of the checkbox.